src/components/CandidateFilterBar/CandidateFilterBar.tsx

```diff
@@ -185,7 +185,7 @@ export const CandidateFilterBar = ({ loading, children }: CandidateFilterBarProp
                     label={tr('Hire streams')}
                     placeholder={tr('Search')}
                     value={hireStreamIdsLocal?.map((i) => String(i))}
-                    items={hireStreams.map((tag) => ({ id: String(tag.id), name: tag.name }))}
+                    items={hireStreams.map((hireStream) => ({ id: String(hireStream.id), name: hireStream.name }))}
```

Comment:
It would be better to use memo for values like this

Reply:
hireStreams goes from `trpc.hireStreams.suggestions.useQuery(...` - should I use memo for this query?

Reply:
for items

Reply:
for items